### PR TITLE
Finish UI of Custom Page

### LINKED ISSUE #1121: Finish off this PR - Front End UI
This PR https://github.com/rubyforgood/homeward-tails/pull/993/files is close to being done, but needs finishing. To make changes, you will need to pull down the branch locally, then make a PR to merge into the branch (like this one https://github.com/rubyforgood/homeward-tails/pull/1103).

- I don't think there is any value showing the images on mobile, so make them show on desktop view only
- The images should be updated to have borders around each section (e.g., hero and about us) like these (but we don't need the side labels): https://www.figma.com/design/0jVgYASUJy0KiX3BVc3dFM/Tasks?node-id=1709-1001&t=mgMF60fFNTB5wShw-1
- Please run the images you get from Figma through **https://tinypng.com/** to make the file size small
- It would be nice to have a bit more of a gap between the two sections to make it clearer they are separate pages

image

### PR DESCRIPTION
**Pull request by jp524**
# 🔗 Issue
Closes #1121

# ✍️ Description

Updated UI per issue description.

# 📷 Screenshots/Demos
**Mobile**
mobile

**Desktop medium**
desktop medium

**Desktop wide**
desktop wide

### CODE REVIEW

**Comment by kasugaijin** [comment]
@all-contributors
please add @jp524 for code.

File: app/assets/images/custom_page_about_us.png
Comment (kasugaijin) [inline app/assets/images/custom_page_about_us.png:1]: Can you please run this through https://tinypng.com/ to compress the image? Thank you!

File: app/assets/images/custom_page_adoptable_pet.png
Comment (kasugaijin) [inline app/assets/images/custom_page_adoptable_pet.png:1]: Same for this image https://tinypng.com/

File: app/assets/images/custom_page_hero.png
Comment (kasugaijin) [inline app/assets/images/custom_page_hero.png:1]: And this one https://tinypng.com/

**Review by kasugaijin — COMMENTED** [review]
Looks great thank you! I think we can compress those images so they load faster in production.

**Comment by jp524** [comment]
> Looks great thank you! I think we can compress those images so they load faster in production.

@kasugaijin I had ran the images through https://tinypng.com/ already but maybe the resolution coming out of Figma was too high so it didn't look like it. I'm compressing the images again and it looks like we're saving ~7% now.

**Comment by kasugaijin** [comment]
@jp524 ok thank you.

**Review by kasugaijin — APPROVED** [review]

**Pull request merged**